### PR TITLE
Stefon.simmons/fix site gettingstarted pgs

### PR DESCRIPTION
**Pull request by StefonSimmons**
### What does this PR do?

updates site-region shortcode syntax on en `/getting_started` pages.

### Motivation

As a part of this card: https://datadoghq.atlassian.net/jira/software/projects/WEB/boards/102?selectedIssue=WEB-3279

### Preview

Check that the TOC items are showing for the preview links below
Check that the hyperlinks on the modified files take you to the appropriate place:
 https://docs-staging.datadoghq.com/stefon.simmons/fix-site-gettingstarted-pgs/

Specifically, notice on the `US1-FED` site region that the `Access the Datadog for Government site` TOC items appears on this page now: https://docs-staging.datadoghq.com/stefon.simmons/fix-site-gettingstarted-pgs/getting_started/site/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by aaxzheng — APPROVED** [review]
Non-blocker for merge but when you have the region set to something that isn't US1-FED and have a section outlined, when you swap to US1-FED, both the current section and the reach government section appear active on the ToC.

image

**Comment by urseberry** [comment]
Something got messed up with the link at the bottom of the Site page: 
image

File: content/en/getting_started/site/_index.md [hunk 1]
```diff
@@ -84,14 +84,14 @@ Different Datadog sites may support different functionalities depending on the i
 
 For example, to see the documentation for the Datadog for Government site, select **US1-FED**.
 
-{{< site-region region="gov" >}}
+{{% site-region region="gov" %}}
 
 ## Access the Datadog for Government site
 
-The Datadog for Government site (US1-FED) is meant to allow US government agencies and partners to monitor their applications and infrastructure. For information about the Datadog for Government site's security and compliance controls and frameworks, as well as how it supports FedRAMP, see the [Security page][1].
+The Datadog for Government site (US1-FED) is meant to allow US government agencies and partners to monitor their applications and infrastructure. For information about the Datadog for Government site's security and compliance controls and frameworks, as well as how it supports FedRAMP, see the [Security page][https://www.datadoghq.com/security/].
```
Comment (urseberry) [inline content/en/getting_started/site/_index.md:91]:
```suggestion
The Datadog for Government site (US1-FED) is meant to allow US government agencies and partners to monitor their applications and infrastructure. For information about the Datadog for Government site's security and compliance controls and frameworks, as well as how it supports FedRAMP, see the [Security page](https://www.datadoghq.com/security/).
```

**Review by urseberry — CHANGES_REQUESTED** [review]

File: content/en/getting_started/site/_index.md (hunk 1 above)
Comment (StefonSimmons) [reply content/en/getting_started/site/_index.md]: @urseberry this is fixed. thanks!

**Review by urseberry — APPROVED** [review]

**Pull request merged**